CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
-cmake_minimum_required(VERSION 3.11)
```

Comment:
Please mention this in the changelog, since it is potentially user-visible (though unlikely to break).

Reply:
I added "- Raised the minimum supported CMake version for building CCF to 3.16 (#2946).", specifically mentioning "building CCF", as the requirement doesn't propagate to installations directly. Though you could argue that some more complex hand-written CMake modules like `ccf_app.cmake` are shipped and we're free to use the features of 3.16 there as well. I'm slightly leaning towards removing "for building CCF", even though the requirement isn't enforced during `find_package(ccf)`.

Reply:
It does propagate indirectly because we may end up relying on 3.16 features, which may leak out via ccf_app.cmake indeed, so I think that's a good way to phrase it.